scripts/tools/nrfconnect/nrfconnect_factory_data.schema

```diff
@@ -109,12 +113,14 @@
         "spake2_salt": {
             "description": "A key-derivation function for the Symmetric Password-Authenticated Key Exchange.",
             "type": "string",
+            "pattern:": "^hex:{1}",
             "minLength": 36,
             "maxLength": 68
         },
         "spake2_verifier": {
             "description": "A verifier for the Symmetric Password-Authenticated Key Exchange",
             "type": "string",
+            "pattern:": "^hex:{1}",
```

Comment:
Is that correct? Isn't the pattern supposed to match the entire value, which would require adding some `.*` at the end?

Reply:
I guess that's correct, according to https://json-schema.org/understanding-json-schema/reference/regular_expressions.html#regular-expressions . `^` means the specific regular expression must be placed at the beginning of the string, and `{1}` means at most only one occurrence.

Reply:
Oh, I see now what you meant. I agree the following part should contain only 0-9, A-F, and a-f symbols. I will add this one: ^hex:{1}([0-9A-Fa-f]){1,} and now after the "hex:" prefix there will be only a proper string that contains specific symbols.

Reply:
Right, also `{1}` is rather redundant :)